config.json.example

```diff
@@ -25,5 +25,9 @@
     "sleeplogReaction": "🛏️",
     "cotwVow": "⚔",
     "cotwReflection": "🛡"
+    "pinMessage": "📌"
+  },
+  "pinLimit": {
+  "pin": 5
   }
```

Comment:
Is there a reason that this isn't just "pinLimit": 5? Does it need the pin child?